tests/containers/netavark_integration.pm

```diff
@@ -19,6 +19,13 @@ my $test_dir = "/var/tmp";
 my $netavark = "";
 my $netavark_version = "";
 
+sub install_ncat {
+    my $ncat_version = "7.95-2";
```

Comment:
IMHO this should be an openQA variable. Hard coding versions is really a bad practice that is going to backfire to us in the future

Reply:
I think in this case this is ok, because this is anyways just a temporary workaround, right @ricardobranco777 ?

Reply:
> IMHO this should be an openQA variable. Hard coding versions is really a bad practice that is going to backfire to us in the future

Agree as it's failing on 15-SP4 due to:

```
Retrieving https://nmap.org/dist/ncat-7.95-2.x86_64.rpm
error: Failed dependencies:
	libc.so.6(GLIBC_2.33)(64bit) is needed by ncat-2:7.95-2.x86_64
	libc.so.6(GLIBC_2.34)(64bit) is needed by ncat-2:7.95-2.x86_64
```